index.html

```diff
@@ -1368,7 +1368,13 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/lizzette-e-advice.html" target="_blank"> Lizzette's Advice</a><br>
       <a href="advice/daniel-mott-advice.html" target="_blank">Daniel M's advice</a><br>
       <a href="advice/kele-bazinga-advice.html"> Kele Bazinga's Advice</a><br>
-      <a href="advice/sharee-stark-advice.html"> Shareee Stark's Advice</a><br>
+      <a href="advice/sharee-stark-advice.html"> Shareee Stark's Advice</a><br>      
+<<<<<<< HEAD
```

Comment:
Hi @kayharr,

Well done! This is nearly there!

There are a few conflict markers remaining in your code. You can see the git diff [here](https://github.com/skillcrush/git-practice/pull/1223/files). 

Once you remove them (while leaving any other student's links), add and commit the changes and then push to GitHub. Your pull request will be updated automatically.

Let us know if you have any questions!

Best,
Lisa